scenarioo-server/src/main/resources/config-for-demo/config.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <configuration>
-	<testDocumentationDirPath>/home/scenarioo/scenarioo/scenarioo-docu-generation-example/build/scenarioDocuExample</testDocumentationDirPath>
```

Comment:
@mi-we there is another config.xml to do the same change in, isnt it?
Update: I fixed this for you.

Reply:
done